test/org/sagebionetworks/bridge/validators/StudyParticipantValidatorTest.java

```diff
@@ -126,6 +126,18 @@ public void emailPhoneOrExternalIdRequired() {
         assertValidatorMessage(validator, withEmail(null), "StudyParticipant", "email, phone, or externalId is required");
     }
     
+    @Test
+    public void emailCannotBeBlank() {
+        validator = new StudyParticipantValidator(externalIdService, study, true);
+        assertValidatorMessage(validator, withEmail(""), "StudyParticipant", "email, phone, or externalId is required");
+    }
```

Comment:
Also, to ensure EMAIL_VALIDATOR catches it, we should test both empty strings and strings of all whitespace.